includes/notifications/controllers/class.llms.notification.controller.quiz.passed.php

```diff
@@ -50,6 +57,62 @@ public function action_callback( $student_id = null, $quiz_id = null ) {
 
 	}
 
+	/**
+	 * Get an array of LifterLMS Admin Page settings to send test notifications
+	 * @param    string     $type  notification type [basic|email]
+	 * @return   array
+	 * @since    [version]
+	 * @version  [version]
+	 */
+	public function get_test_settings( $type ) {
+
+		if ( 'email' != $type ) {
+			return;
+		}
+
+		$query = new LLMS_Query_Quiz_Attempt( array(
+			'per_page' => 25,
+			'attempt' => true,
+		));
+
+		$options = array(
+		   '' => '',
+		);
+		$attempts = array();
+		$results = $query->results;
+
+		if ( $results ) {
+			foreach ( $results as $result ) {
+				$attempts[] = new LLMS_Quiz_Attempt( $result->id );
+			}
+		}
+
+		foreach ( $attempts as $attempt ) {
+			$quiz = llms_get_post( $attempt->get( 'quiz_id' ) );
+			$student = llms_get_student( $attempt->get( 'user_id' ) );
+			if ( $attempt && $student ) {
+				$options[ $attempt->get( 'id' ) ] = esc_attr( sprintf( __( 'Attempt #%1$d for Quiz "%2$s" by %3$s', 'lifterlms' ), $attempt->get( 'id' ), $quiz->get( 'title' ), $student->get_name() ) );
+			}
+		}
+
+		return array(
```

Comment:
Looks like theres spaces in this array instead of tabs, see output of PHPCS at https://travis-ci.org/gocodebox/lifterlms/jobs/436806977#L1198